Source/Blazorise/Components/DatePicker/DatePicker.razor.cs

```diff
@@ -477,6 +477,17 @@ private object GetLocalizationObject()
     /// <inheritdoc/>
     protected override bool IsSameAsInternalValue( IReadOnlyList<TValue> value ) => value.AreEqual( InternalValue );
 
+    /// <inheritdoc/>
+    protected override string GetFormatedValueExpression()
+    {
+        if ( DateExpression is null )
```

Comment:
You created an issue to rename stuff to **Value**. **Expression** should be included I guess. If Expression is somehow centralized like Expression<Func<TValue>> something along these lines, then we might not need to override this per component, and just do it on base?

Reply:
Exactly. That is the plan with `ValueExpression`. All these overrides are needed because we have have too many different APIs. With `Value`, we will centralize it in the base input component.